tools/flang1/flang1exe/lowersym.c

```diff
@@ -2753,7 +2762,11 @@ lower_put_datatype(int dtype, int usage)
     putsym("member", DTY(dtype + 1));
     putval("size", DTY(dtype + 2));
     putsym("tag", DTY(dtype + 3));
-    putval("align", DTY(dtype + 4));
+
+    if (DTA(dtype) > DTY(dtype + 4))
```

Comment:
Comment?

Reply:
> Comment?

This code is used to export alignment values to the .stb file. I have refactored this section of the code and added the necessary comment to the import code part.